tests/unit/client_test.ts

```diff
@@ -75,18 +78,21 @@ for (const browserItem of browserList) {
           await promise;
           await browser.close();
         },
+        ignore: remote //Ignoring as binary path is not a necessisty to test for remote browsers
```

Comment:
i'd suggest:

1. remove all these `ignore: remote`s
2. remove the `waiter` method
3. Have your new conditional that gets the ws url, pol the endpoint until its ready, and timeout after 5s

Reason for this is, if someone were to use sinco, theyd also have to create a waiting method
5. On line

Reply:
I agree on the waiter part...
on the ignore part, I will try to come up with a docker network such that execution happens on a seperate container and browser (and the drash server) is on another container..

Reply:
and yes also, The binary path, i think that can be skipped for remote tests, because generally for remote executions, a running browser instance is expected rather than the path to the remote binary file

Reply:
yeah i agree with your last point :)

Reply:
Have added a way for the remote container browser to navigate to host server